src/animations/AnimationManager.js

```diff
@@ -323,7 +323,7 @@ var AnimationManager = new Class({
      * @property {integer} [end=-1] - [description]
      * @property {boolean} [first=false] - [description]
      * @property {AnimationFrameConfig[]} [outputArray=[]] - [description]
-     * @property {boolean} [frames=false] - [description]
+     * @property {boolean|number[]} [frames=false] - [description]
```

Comment:
Please wrap the values: `{(boolean|integer[])}` (integer makes more sense as the type here, as a float is invalid)